### PR TITLE
updated data_module folder

### PR DESCRIPTION
**Pull request by haripriya9647**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Relevant Issue
Issue #276  the issue here

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by divyanshugit** [comment]
In this PR, please try to write the whole inline documentation inside the class. You can explain about methods inside it

File: fedrec/data_models/envis_module.py [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 @Registrable.register_class_ref
 class EnvisModule(Serializable):
+    """
+    Envismodule is the base module for pytorch modules.
+    All other pytorch modules when called will have
+    EnvisModule as base which is used to serialize and
+    decirialize data objects.
```
Comment (divyanshugit) [inline fedrec/data_models/envis_module.py:15]: Please rephrase it. It's kind of confusing. Add small pieces of documentation for methods as well.

Since It's a special directory in this `fedrec` module. Please try to write the documentation by examples with text. You can take inspiration from [here](https://pytorch.org/docs/stable/tensors.html#torch.Tensor).

File: fedrec/data_models/state_tensors_model.py [hunk 2]
```diff
@@ -7,6 +7,23 @@
 
 @Registrable.register_class_ref
 class StateTensors(EnvisTensors):
+    """
+    The StateTensors class majorly focuses on naming
+    tensor states and their paths, with serialization
+    and deserialization methods implemented.
```
Comment (divyanshugit) [inline fedrec/data_models/state_tensors_model.py:13]: Same as above

File: fedrec/data_models/tensors_model.py [hunk 3]
```diff
@@ -9,7 +9,20 @@
 
 @Registrable.register_class_ref
 class EnvisTensors(Serializable):
-
+    """
+    The EnvisTensors class majorly focuses on naming
+    tensor states ,their paths,collecting torch object
+    with serialization and deserialization methods
+    implemented.
```
Comment (divyanshugit) [inline fedrec/data_models/tensors_model.py:16]: Same as above

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/data_models/tensors_model.py [hunk 4]
```diff
@@ -9,7 +9,20 @@
 
 @Registrable.register_class_ref
 class EnvisTensors(Serializable):
-
+    """
+    The EnvisTensors class majorly focuses on naming
+    tensor states ,splitting their paths and collecting
+    torch objects with the help of implementation of
+    serialization and deserialization methods.
```
Comment (divyanshugit) [inline fedrec/data_models/tensors_model.py:16]: Please utilize the 79 letters per line

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/data_models/envis_module.py [hunk 5]
```diff
@@ -48,10 +55,18 @@ def envis_state(self):
...
+        Loads the envis state dict.
+        '''
         self.original_reference.load_state_dict(state)
 
     def serialize(self):
         # TODO decide how to fill storage from config
+        '''
+        Serialize refers to the process of converting a data
+        object into a format that allows us to store or
+        transmit the data.
```
Comment (divyanshugit) [inline fedrec/data_models/envis_module.py:68]: Please utilize the 70 letters in each file.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
Please utilize the 70 letters in each file.

**Comment by haripriya9647** [comment]
@divyanshugit i have added commits as per suggestions.Please review
